### PR TITLE
split diff report into multiple pages

### PR DESCRIPTION
**Pull request by js-xc**

### CODE REVIEW

File: config/diffreportgenerator.properties [hunk 1]
```diff
@@ -36,3 +36,43 @@ appender.file.strategy.type = DefaultRolloverStrategy
...
+#
+# XSL Transformations
+#
+# After generating the raw result data files, a couple of XSL transformations
+# are performed to get viewable HTML report pages. These transformations are
+# configured now.
+#
+# Format:
+#
+#   com.xceptance.xlt.reportgenerator.transformations.<idx>.<styleSheetFileName|outputFileName>
```
Comment (jowerner) [inline config/diffreportgenerator.properties:50]:
```suggestion
#   com.xceptance.xlt.diffreportgenerator.transformations.<idx>.<styleSheetFileName|outputFileName>
```

File: config/xsl/trendreport/transactions.xsl [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
+
+<xsl:output method="html" 
+            indent="no"
```
Comment (jowerner) [inline config/xsl/trendreport/transactions.xsl:5]:
```suggestion
            indent="yes" 
```

Please do that also in `requests.xsl`.

File: config/xsl/trendreport/sections/navigation.xsl [hunk 3]
```diff
@@ -5,7 +5,7 @@
         <nav>
```
Comment (jowerner) [inline config/xsl/trendreport/sections/navigation.xsl:5]: Do we have to add the responsive nav stuff here as well?

**Comment by jowerner** [comment]
Please go through all new and existing top-level stylesheet files of diff and trend report and check/add/fix the respective browser tab title suffix, such as "... - Transactions" for the transactions page and  "... - Overview" for the index page.

**Comment by jowerner** [comment]
While we are at `config/xsl/diffreport/sections/navigation.xsl`, can we clean it up a little? Thanks!
* Remove `<xsl:param name="current" />`.
* Remove `<xsl:text />`.
* Remove the `CDATA` section.
* Remove the sub menu of the Overview nav entry. It is not really helpful.

**Review by jowerner — CHANGES_REQUESTED** [review]
See comments on the Conversation tab.

File: config/xsl/trendreport/sections/navigation.xsl (hunk 3 above)
Comment (js-xc) [reply config/xsl/trendreport/sections/navigation.xsl:5]: Will do.

**Pull request merged**